flipt-client-python/flipt_client/models.py

```diff
@@ -1,6 +1,8 @@
 from pydantic import BaseModel
 from typing import List, Optional
 
+import pydantic
```

Comment:
```suggestion
from pydantic import BaseModel, RootModel
from typing import List, Optional
```

Reply:
i tried that previously and it was giving me troubles about it couldn't find that import, will try again tho